src/main/java/com/microsoft/azure/vmagent/AzureVMAgentTemplate.java

```diff
@@ -126,6 +125,8 @@
 
     private String subnetName;
 
+    private boolean usePrivateIp;
+
```

Comment:
nothing serious ... would be nice to see the name of the variable ends with "IP", but you can ignore it